rnarch/src/screens/login/LoginScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text } from 'atoms/Text';
+import { Box } from 'atoms/Box';
+import { TextInput } from 'atoms/TextInput';
+import { Button } from '/Users/srishtimaurya/Documents/Crownstack/training /architecture/practise/rnarch/src/molecules/Button/Button.tsx';
+import { StyleSheet } from 'react-native';
+export const LoginScreen = () => {
+  return (
+    <Box style={styles.container}>
```

Comment:
You can write like this  <Box width="90%" alignSelf="center" mt="xl" bg='aliceBlue'>